src/main/java/com/pmm/budgetcalculator/expensing/exeption/ExpenseCategoryNotFoundException.java

```diff
@@ -0,0 +1,8 @@
+package com.pmm.budgetcalculator.expensing.exeption;
+
+public class ExpenseCategoryNotFoundException extends RuntimeException {
```

Comment:
This is quite unnecessary. Given you will have 100 entities or even 10000 - will you create xxxNotFoundException for each one? What about some generalization or abstraction? EntityNotFound maybe...?